dags/openshift_nightlies/config/install/aws/ovn-large-cp.json

```diff
@@ -18,7 +18,7 @@
     "openshift_worker_root_volume_iops": 0,
     "machineset_metadata_label_prefix": "machine.openshift.io",
     "openshift_infra_node_instance_type": "m5.12xlarge",
-    "openshift_workload_node_instance_type": "m5.4xlarge",
+    "openshift_workload_node_instance_type": "m5.2xlarge",
```

Comment:
m5.2xlarge is good enough for benchmark-operator pods and kube-burner/MB pod?

Reply:
This is the control-plane dag config, no MB here. kube-burner would have enough resources.